src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_4_horizontal_layout.html.twig

```diff
@@ -14,6 +14,8 @@
     {%- endif -%}
 {%- endblock form_label %}
 
+{%- block form_label_errors "" -%}
```

Comment:
Do we use the short block syntax anywhere else? Otherwise I suggest to not introduce that here. But 👍 from me for the changes here.

Reply:
Nice catch. I only found one usage in some Tests for the short syntax, but several usages of the explicit syntax in some templates.
So I changed it to the longer version.